### ISSUE TITLE
Type of losses supported 

### ISSUE THREAD

**Issue by cspearl** [open-mmlab/mmsegmentation#1950]
Hi I just wanted to know what are the types of losses that are supported other than CrossEntropy? Can the class weights be assigned for any loss function that I use?

**Comment by xiexinch** [comment]
Hi @cspearl,
You might find them in the `losses` folder.
https://github.com/open-mmlab/mmsegmentation/tree/master/mmseg/models/losses
The `class_weight` is supported by each loss function we implemented.

**Comment by cspearl** [comment]
Thanks
